src/components/Actions/actions.js

```diff
@@ -0,0 +1,100 @@
+import axios from "axios";
+
+export const FETCHING_GET_PLANT = 'FETCHING_GET_PLANT';
+export const FETCHING_GET_PLANT_SUCESS = 'FETCHING_ADD_STAR_SUCESS';
+export const FETCHING_GET_PLANT_FAILURE = 'FETCHING_ADD_STAR_SUCESS';
+
+export const FETCHING_UPDATE_PLANT = 'FETCHING_UPDATE_PLANT';
+export const FETCHING_UPDATE_PLANT_SUCESS = 'FETCHING_ADD_STAR_SUCESS';
+export const FETCHING_UPDATE_PLANT_FAILURE = 'FETCHING_ADD_STAR_SUCESS';
+
+export const FETCHING_REMOVE_PLANT = 'FETCHING_REMOVE_PLANT_';
+export const FETCHING_REMOVE_PLANT_SUCESS = 'FETCHING_ADD_STAR_SUCESS';
+export const FETCHING_REMOVE_PLANT_FAILURE = 'FETCHING_ADD_STAR_SUCESS';
```

Comment:
You have `FETCHING_ADD_STAR_SUCESS` and `FETCHING_ADD_STAR_SUCESS` for all these. This should be corrected before merging